package.json

```diff
@@ -148,7 +148,7 @@
     "style-loader": "^0.19.0",
     "uglifyjs-webpack-plugin": "^1.0.0",
     "unused-files-webpack-plugin": "^3.0.0",
-    "webpack": "^3.8.1",
+    "webpack": "^5.37.0",
```

Comment:
# Upgrade Webpack to the latest version
`yarn upgrade webpack@latest`
```
warning " > @cypress/webpack-preprocessor@4.1.5" has incorrect peer dependency "webpack@^4.18.1".
warning " > @testing-library/cypress@5.3.1" has incorrect peer dependency "cypress@^2.1.0 || ^3.0.0 || ^4.0.0".
warning " > babel-loader@7.1.5" has incorrect peer dependency "webpack@2 || 3 || 4".
warning " > extract-text-webpack-plugin@3.0.2" has incorrect peer dependency "webpack@^3.1.0".
warning " > html-webpack-plugin@2.30.1" has incorrect peer dependency "webpack@1 || ^2 || ^2.1.0-beta || ^2.2.0-rc || ^3".
warning " > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
warning " > uglifyjs-webpack-plugin@1.3.0" has incorrect peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
warning " > webpack-dev-server@2.11.5" has incorrect peer dependency "webpack@^2.2.0 || ^3.0.0".
warning "webpack-dev-server > webpack-dev-middleware@1.12.2" has incorrect peer dependency "webpack@^1.0.0 || ^2.0.0 || ^3.0.0".
```

Running `yarn-dev-ee` after this produces the following error:
```
[frontend] internal/modules/cjs/loader.js:883
[frontend]   throw err;
[frontend]   ^
[frontend] 
[frontend] Error: Cannot find module 'webpack/bin/config-yargs'
[frontend] Require stack:
[frontend] - /Users/offbynone/code/work/metabase/node_modules/webpack-dev-server/bin/webpack-dev-server.js
[frontend]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
[frontend]     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
[frontend]     at Module.require (internal/modules/cjs/loader.js:952:19)
[frontend]     at require (internal/modules/cjs/helpers.js:88:18)
[frontend]     at Object.<anonymous> (/Users/offbynone/code/work/metabase/node_modules/webpack-dev-server/bin/webpack-dev-server.js:54:1)
[frontend]     at Module._compile (internal/modules/cjs/loader.js:1063:30)
[frontend]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
[frontend]     at Module.load (internal/modules/cjs/loader.js:928:32)
[frontend]     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
[frontend]     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
[frontend]   code: 'MODULE_NOT_FOUND',
[frontend]   requireStack: [
[frontend]     '/Users/offbynone/code/work/metabase/node_modules/webpack-dev-server/bin/webpack-dev-server.js'
[frontend]   ]
[frontend] }
error Command failed with exit code 1.
```

Searching online for a solution:
- https://stackoverflow.com/a/64205610/8815185
> If you're using webpack-cli 4 or webpack 5, change webpack-dev-server to webpack serve.

The next logical step: upgrade `webpack-dev-server`

Reply:
Test comment